homeassistant/components/octoprint/strings.json

```diff
@@ -32,5 +32,29 @@
     "progress": {
       "get_api_key": "Open the OctoPrint UI and click 'Allow' on the Access Request for 'Home Assistant'."
     }
+  },
+  "services": {
+    "printer_connect": {
+      "name": "Connect to a printer",
+      "description": "Instructs the octoprint server to connect to a printer.",
+      "fields": {
+        "device_id": {
+          "name": "Server",
+          "description": "The server that should connect."
+        },
+        "profile_name": {
+          "name": "Profile Name",
```

Comment:
```suggestion
          "name": "Profile name",
```

Reply:
I don't see this as a showstopper for this PR. Unless there's some coding standard this violates for HA.